File: Front/src/app/Models/Imc.ts
import { Usuario } from "./usuario.model";

export interface Imc {
  id?: number;
  nomeAId: number;
  nomeA?: Usuario;
  nascimentoAId?: Usuario;
  altura?: number;
  peso?: number;
  calculoImc: number;
  classificacao: string;
  criadoEm: string;
}

File: Back/Models/Imc.cs
using System;

namespace API.Models
{
    public class Imc{
        public Imc() => CriadoEm = DateTime.Now;
        public string NomeAId { get; set; }
        public string NomeA { get; set; }
        public string NascimentoAId { get; set; }
         public double Altura { get; set; }
         public double Peso { get; set; }
         public double CalculoImc { get; set; }
        public string Classificacao { get; set; }
        public DateTime CriadoEm { get; set; }
    }
} 